dts/arm/atmel/sam4s.dtsi

```diff
@@ -19,12 +19,11 @@
 	};
 
 	sram0: memory {
-		compatible = "sram";
+		compatible = "mmio-sram";
 		reg = <0x20100000 0x20000>;
 	};
 
 	flash0: flash {
-		compatible = "flash";
```

Comment:
They flash nodes don't currently have a yaml associated with them.  We need to see if the flash/mtd support in linux from a dt point of view makes sense to use or model off of.  We need to introduce a flash yaml at some point.